UKHomeOffice/cerberus-service
# COP-10462: Add tests to verify Itinerary details of airpax task

## Description
Add tests to verify Itinerary details of airpax task

## To Test
npm run cypress:runner
run the test from spec `airpax-task-details.spec.js` 

`Should verify Itinerary details of an AirPax task on task details page`

## Developer Checklist

\* Required
